cdk-deployment/validator_cdk/validator_cdk_stack.py

```diff
@@ -13,17 +13,17 @@ class ValidatorCdkStack(cdk.Stack):
     def __init__(self, scope: cdk.Construct, construct_id: str, **kwargs) -> None:
         super().__init__(scope, construct_id, **kwargs)
 
-        build_path = Path("../")
+        build_path = Path("./lambda")
         print(str(build_path.resolve()))
 
         # Defines an AWS Lambda resource
         validator_lambda = _lambda.Function(
             self,
-            "STACValidator",
+            f"{construct_id}-lambda",
```

Comment:
Where does construct_id come from? I don't see you passing it to the constructor. Also, when Cloudformation/CDK create names for resources, it already uses something like this. I usually like to add the stage/env name to this id, so they are distinguishable from each other.